### PR TITLE
Merge Fix SwitchWorkspace not properly loading saved data

### LINKED ISSUE #130: Switching workspace overwrites data that has been loaded in

### PR DESCRIPTION
**Pull request by EyoWeiChin**
closes #130

### CODE REVIEW

**Review by ChanJianHao — APPROVED** [review]
Looks good to merge!

File: src/main/java/seedu/duke/command/SwitchWorkspaceCommand.java [hunk 1]
```diff
@@ -26,10 +25,9 @@ public SwitchWorkspaceCommand() {
     @Override
     public String execute(AnimeData animeData, StorageManager storageManager, User user) throws AniException {
         //Find the user and setActiveUser to it
-        Workspace chgWorkspace = user.getWorkspace(switchToThisWorkspace);
-        user.setActiveWorkspace(chgWorkspace);
-        String result = SWITCH_SUCCESS_HEADER + switchToThisWorkspace;
-        return result;
+        //Workspace chgWorkspace = user.getWorkspace(switchToThisWorkspace);
```
Comment (OngDeZhi) [inline src/main/java/seedu/duke/command/SwitchWorkspaceCommand.java:28]: Do you still need this? Otherwise, it would be better to remove the old code.

Comment (OngDeZhi) [inline src/main/java/seedu/duke/command/SwitchWorkspaceCommand.java]: Do you still need this? Otherwise, it would be better to remove the old code.

Comment (EyoWeiChin) [reply src/main/java/seedu/duke/command/SwitchWorkspaceCommand.java]: Ah, I left it there mistakenly. I have fixed it, thank you for pointing it out.

**Review by OngDeZhi — APPROVED** [review]
LGTM :)

**Pull request merged**